README.md

```diff
@@ -17,216 +48,361 @@ See also:
-
-## Manual installation on the k3d cluster
+Run the following commands to deploy Keda Manager on a target k8s cluster (i.e on k3d):
 
 1. Clone the project.
 
    ```bash
    git clone https://github.com/kyma-project/keda-manager.git && cd keda-manager/
    ```
 
-2. Provide the k3d cluster.
+2. Set the `keda-manager` image name.
 
+   > NOTE: You can use local k3d registry or your dockerhub account to push intermediate images.  
    ```bash
-   kyma provision k3d
+   export IMG=<DOCKER_USERNAME>/custom-keda-manager:0.0.1
    ```
 
-3. Build and push the Keda Manager image.
+3. Verify the compatibility.
 
    ```bash
-   make module-image IMG_REGISTRY=localhost:5001/unsigned/operator-images IMG=localhost:5001/keda-manager-dev-local:0.0.1
+   make test
    ```
-
-4. Build and push the Keda module.
+4. Build and push the image to the registry.
 
    ```bash
-   make module-build IMG=k3d-kyma-registry:5001/keda-manager-dev-local:0.0.1 MODULE_REGISTRY=localhost:5001/unsigned
+   make module-image
    ```
-
-5. Verify if the module and the manager's image are pushed to the local registry.
+5. Create a target namespace.
 
    ```bash
-   curl localhost:5001/v2/_catalog
+   kubectl create ns kyma-system
    ```
-You should get a result similar to this example:
 
-   ```json
-   {"repositories":["keda-manager-dev-local","unsigned/component-descriptors/kyma-project.io/module/keda"]}
-   ```
-6. Inspect the generated module template.
+6. Deploy.
 
-> **NOTE:** The following sub-steps are temporary workarounds.
-
-Edit `template.yaml` and:
-- change `target` to `control-plane`
-
-   ```yaml
-   spec:
-    target: control-plane
-    ```
-> **NOTE:** This is required in the single-cluster mode only.
+   ```bash
+   make deploy
+   ```
 
-- change the existing repository context in `spec.descriptor.component`:
+7. Verify if keda manager is deployed
 
-   ```yaml
-   repositoryContexts:      
-     - baseUrl: k3d-kyma-registry.localhost:5000/unsigned
-       componentNameMapping: urlPath
-       type: ociRegistry
+   ```bash
+   kubectl get deployments -n kyma-system       
+   NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+   operator-keda-manager            1/1     1            1           1m
    ```
 
-> **NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 5001. K3d has registry aliases, but `module-manager` is not part of k3d and does not know how to properly alias k3d-kyma-registry.localhost:5001.
 
-7. Install the modular Kyma on the k3d cluster.
+### How to use Keda Manager to instal Keda
 
-> **NOTE** This installs the latest versions of `module-manager` and `lifecycle-manager`.
+Keda manager installs KEDA based on watched Keda Custom Resources (CR):
 
-Use the `--template` flag to deploy the Keda module manifest from the beggining, or apply it using kubectl later.
+- Apply Keda CR (sample) to have Keda installed.
```

Comment:
```suggestion
- Apply Keda CR (sample) to install Keda.
```

Reply:
I actually think the passive voice is better here as its not the user uninstalling it, but manager reacting to CR removal